packages/flutter/lib/src/material/stepper.dart

```diff
@@ -0,0 +1,648 @@
+                // In the vertical case we need to scroll to the newly tapped
+                // step.
+                Scrollable.ensureVisible(
+                  _keys[i].currentContext,
+                  curve: Curves.fastOutSlowIn,
+                  duration: kThemeAnimationDuration
+                );
+
+                if (config.onStepTapped != null)
+                  config.onStepTapped(i);
+              } : null,
+              child: _buildVerticalHeader(context, i)
+            ),
+            _buildVerticalBody(i)
+          ]
+        )
+      );
+    }
+
+    return new Block(
+      children: children
+    );
+  }
+
+  Widget _buildHorizontal(BuildContext context) {
+    List<Widget> children = <Widget>[];
+
+    for (int i = 0; i < config.steps.length; i += 1) {
+      children.add(
+        new InkResponse(
+          onTap: config.steps[i].state != StepState.disabled ? () {
+            if (config.onStepTapped != null)
+              config.onStepTapped(i);
+          } : null,
+          child: new Row(
+            children: <Widget>[
+              new Container(
+                height: 72.0,
+                child: new Center(
+                  child: _buildIcon(context, i)
+                )
+              ),
+              new Container(
+                margin: const EdgeInsets.only(left: 12.0),
+                child: _headerText(i)
+              )
+            ]
+          )
+        )
+      );
+
+      if (!_last(i))
+        children.add(
+          new Flexible(
+            child: new Container(
+              margin: const EdgeInsets.symmetric(horizontal: 8.0),
+              height: 1.0,
+              decoration: new BoxDecoration(
+                backgroundColor: Colors.grey[400]
+              )
+            )
+          )
+        );
+    }
+
+    return new Column(
+      children: <Widget>[
+        new Material(
+          elevation: 2,
+          child: new Container(
+            margin: const EdgeInsets.symmetric(horizontal: 24.0),
+            child: new Row(
+              children: children
+            )
+          )
+        ),
+        new Flexible(
+          child: new ScrollableViewport(
+            child: new Container(
+              margin: const EdgeInsets.all(24.0),
+              child: new Column(
+                children: <Widget>[
+                  new AnimatedSize(
+                    curve: Curves.fastOutSlowIn,
+                    duration: kThemeAnimationDuration,
+                    child: config.steps[config.currentStep].content
+                  ),
+                  _buildVerticalControls()
+                ]
+              )
+            )
+          )
+        )
+      ]
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    assert(debugCheckHasMaterial(context));
+    assert(() {
+      if (context.ancestorWidgetOfExactType(Stepper) != null)
+        throw new FlutterError('Steppers cannot be nested.');
```

Comment:
Should make this a multiline message. At the end of your current string add a '\n', and then add some text, maybe "The material specification advises that one should avoid embedding steppers within steppers."